include/RAJA/config.hpp.in

```diff
@@ -593,6 +593,20 @@ T * align_hint(T * x)
 #endif
 }
 
+// Internal variable to toggle Caliper profiling on and off
+// Users have access to a global function.
+namespace expt {
+  namespace detail {
+    inline bool RAJA_caliper_profile = false;
```

Comment:
Is this the preferred pattern for globals now that we can use c++17? I've always leaned towards defining a static variable in a function to avoid the possibility of static initialization order bugs. The CaliperPlugin is also initialized pre-main and it sets this variable so it seems possible that there could be issues.